src/games/strategy/engine/data/ChangeFactory.java

```diff
@@ -201,26 +185,10 @@ public static Change genericTechChange(final TechAttachment attachment, final Bo
     return new GenericTechChange(attachment, value, property);
   }
 
-  public static Change changeGameSteps(final GameSequence oldSequence, final GameStep[] newSteps) {
```

Comment:
I am not sure if this one (`changeGameSteps` and `GameSequenceChange`) should be kept or not.  This may be good functionality to make available to map makers?  Or it may introduce too many problems to let people start using it...